src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.SymbolToProjectId.cs

```diff
@@ -62,31 +63,18 @@ internal partial class SolutionState
                     // Have to lock as there's no atomic AddOrUpdate in netstandard2.0 and we could throw if two
                     // threads tried to add the same item.
 #if NETSTANDARD
```

Comment:
This seems a bit clearer, since you don't need to know that .NET Framework isn't a compilation target:

```suggestion
#if !NETCOREAPP
```